plugin/aws/s3Dir.go

```diff
@@ -38,10 +38,10 @@ func (s *s3Dir) List(ctx context.Context) ([]plugin.Entry, error) {
 
 	buckets := make([]plugin.Entry, len(resp.Buckets))
 	for i, bucket := range resp.Buckets {
-		name := bucket.Name
+		name := awsSDK.StringValue(bucket.Name)
 
 		locRequest := &s3Client.GetBucketLocationInput{
-			Bucket: name,
+			Bucket: awsSDK.String(name),
```

Comment:
How about use `bucket.Name` directly here so you don't have to cast twice?

Reply:
I remember running into some weird things when I did that for other values, but since this code worked OK before, I can go back to re-using `bucket.Name`.